examples/3-mutation/src/Dog.re

```diff
@@ -62,9 +62,11 @@ let make =
       [|id|],
     );
 
+  // Example of using hooks with graphql_ppx
```

Comment:
I think we should mention both PPXs in case someone stumbles across this and thinks the lib only works with `graphql_ppx`.

```suggestion
  /* Example of using hooks with graphql_ppx / graphql_ppx_re. */
```

Reply:
Do you prefer non-single-line comments?

Reply:
I think I just tend to use `/* */` mostly out of habit from the Reason docs, I actually didn't know that single line comments were supported. Happy either way!